build/assets/hypothesis.html

```diff
@@ -0,0 +1,2 @@
+<!-- Enable Hypothesis annotation. https://web.hypothes.is/ -->
+<script src="https://hypothes.is/embed.js" async></script>
```

Comment:
@cgreene note that this update will enable Hypothesis annotation on the deep review HTML manuscript.